# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gpsdrive/gnome2.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gpsdrive-2.10pre6-orig/cmake/Modules/FindGTK2.cmake gpsdrive-2.10pre6/cmake/Modules/FindGTK2.cmake
--- gpsdrive-2.10pre6-orig/cmake/Modules/FindGTK2.cmake	2009-02-09 07:09:21.000000000 +0000
+++ gpsdrive-2.10pre6/cmake/Modules/FindGTK2.cmake	2009-02-09 07:11:14.000000000 +0000
@@ -89,7 +89,7 @@
       ${_GTK2_INCLUDEDIR}
       /usr/include/gtk-2.0
       /usr/local/include/gtk-2.0
-      /opt/gnome/include/gtk-2.0
+      /opt/gnome2/include/gtk-2.0
     )
 
     # Some Linux distributions (e.g. Red Hat) have glibconfig.h
@@ -106,11 +106,11 @@
       endif (PKG_CONFIG_FOUND)
     endif (${CMAKE_MAJOR_VERSION} EQUAL 2 AND ${CMAKE_MINOR_VERSION} EQUAL 4)
 
-    SET(GDIR /opt/gnome/lib/glib-2.0/include)
+    SET(GDIR /opt/gnome2/lib/glib-2.0/include)
     FIND_PATH(GTK2_GLIBCONFIG_INCLUDE_PATH glibconfig.h
       ${_GLIB2_INCLUDEDIR}
-      /opt/gnome/lib64/glib-2.0/include
-      /opt/gnome/lib/glib-2.0/include
+      /opt/gnome2/lib64/glib-2.0/include
+      /opt/gnome2/lib/glib-2.0/include
       /usr/lib64/glib-2.0/include
       /usr/lib/glib-2.0/include
     )
@@ -118,7 +118,7 @@
 
     FIND_PATH(GTK2_GLIB_INCLUDE_PATH glib.h
       ${_GLIB2_INCLUDEDIR}
-      /opt/gnome/include/glib-2.0
+      /opt/gnome2/include/glib-2.0
       /usr/include/glib-2.0
     )
     #MESSAGE(STATUS "DEBUG: GTK2_GLIBCONFIG_INCLUDE_PATH = ${GTK2_GLIBCONFIG_INCLUDE_PATH}")
@@ -128,7 +128,7 @@
       /usr/include
       /usr/local/include
       /usr/openwin/share/include
-      /opt/gnome/include
+      /opt/gnome2/include
     )
 
     if (${CMAKE_MAJOR_VERSION} EQUAL 2 AND ${CMAKE_MINOR_VERSION} EQUAL 4)
@@ -141,7 +141,7 @@
 
     FIND_PATH(GTK2_PANGO_INCLUDE_PATH pango/pango.h
       ${_PANGO_INCLUDEDIR}
-      /opt/gnome/include/pango-1.0
+      /opt/gnome2/include/pango-1.0
       /usr/include/pango-1.0
     )
 
@@ -155,8 +155,8 @@
 
     FIND_PATH(GTK2_GDKCONFIG_INCLUDE_PATH gdkconfig.h
       ${_GDK2_INCLUDEDIR}
-      /opt/gnome/lib/gtk-2.0/include
-      /opt/gnome/lib64/gtk-2.0/include
+      /opt/gnome2/lib/gtk-2.0/include
+      /opt/gnome2/lib64/gtk-2.0/include
       /usr/lib/gtk-2.0/include
       /usr/lib64/gtk-2.0/include
     )
@@ -171,7 +171,7 @@
 
     FIND_PATH(GTK2_CAIRO_INCLUDE_PATH cairo.h
       ${_CAIRO_INCLUDEDIR}
-      /opt/gnome/include/cairo
+      /opt/gnome2/include/cairo
       /usr/include
       /usr/include/cairo
     )
@@ -187,7 +187,7 @@
 
     FIND_PATH(GTK2_ATK_INCLUDE_PATH atk/atk.h
       ${_ATK_INCLUDEDIR}
-      /opt/gnome/include/atk-1.0
+      /opt/gnome2/include/atk-1.0
       /usr/include/atk-1.0
     )
     #MESSAGE(STATUS "DEBUG: GTK2_ATK_INCLUDE_PATH = ${GTK2_ATK_INCLUDE_PATH}")
@@ -201,7 +201,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_GTK_LIBRARY
@@ -213,7 +213,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_GDK_LIBRARY
@@ -225,7 +225,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_GMODULE_LIBRARY
@@ -237,7 +237,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_GLIB_LIBRARY
@@ -249,7 +249,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_Xi_LIBRARY 
@@ -260,7 +260,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     FIND_LIBRARY(GTK2_GTHREAD_LIBRARY
@@ -271,7 +271,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
 
@@ -283,7 +283,7 @@
         /usr/local/lib
         /usr/openwin/lib
         /usr/X11R6/lib
-        /opt/gnome/lib
+        /opt/gnome2/lib
     )
 
     IF(GTK2_GTK_INCLUDE_PATH)
